workflow/resources/datavzrd/diffexp-template.yaml

```diff
@@ -1,8 +1,10 @@
+__use_yte__: true
+
 name: ?f"Differential expression analysis for model {wildcards.model}"
 datasets:
   genes_representative:
     path: ?input.genes_representative
-    offer-excel: true
+    offer-excel: ?params.offer_excel in ["true", "True"]
```

Comment:
I don't like the check for true and True as strings. The yaml schema should ensure that the value in the config yaml is a boolean. And then, there is no need to have these additional checks here.

Reply:
Yeah, that's a good idea. Actually, I have no clue why the `true` or `false` in this case got parsed into a string, while in the diffsplice case it seems to have turned into a boolean. Neither of them was mentioned in the `config.schema.yaml` so far...